recipe/build.sh

```diff
@@ -3,6 +3,9 @@
 if [ "$(uname)" == "Darwin" ]; then
     export CXX="${CXX} -stdlib=libc++"
     export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+    # Shared Fortran libs on OSX unsupported as of 1.8.18
+    FORTRAN_FLAGS=("--enable-fortran" "--enable-fortran2003")
```

Comment:
@jakirkham Just passing the ```--disable-shared``` flag seems to have done the trick. Would you like me to throw in something like ```libbz2``` to nuke the old shared lib when users upgrade?